moderntribe/tribe-common
# Merge B20.11 into G20.12

- TEC-3612 - get admin fields add to interface and abstract
- TEC-3612 - remove comments on admin form saving
- TEC-3612 - add get_admin_html method, modify use of get arguements, and other code review changes.
- refactor(src) pair-programming
- TEC-3612 & TEC-3613 - Formatting updates and docblock
- TEC-3612 & TEC-3613 - Add in the default arguments to setup
- TEC-3997 & TEC-3620 - Dynamic Data to front end for TEC List Widget
- TEC-3597 - Updates to the filters for Pro to be able to modify the widgets
- Set version to 4.12.13
- Fix typos
- Added a setup_view method to the abstract class. Since it calls it internally and it shouldn't call methods it doesn't define?
- build(.travis.yml) rollback composer to version 1
- Add styles for widget section(s). (See TEC & ECP PRs)
- Common Style sync
- Need to tweak where we load the common admin styles. To include widgets screen.
- Add check for $is_widgets
- Add featured svg icon
- Add Featured icon test
- Add recurring icon
- Add search and location icons
- Add day, list, map, month, photo, week icons
- Remove old featured class
- Remove old search icon class
- Remove old recurring class
- Remove other unused classes
- wip - use currentColor
- Replace loader dots icon
- Add dots icon test
- Update snaps
- Add TBD version and remove fill, stroke, width, height from svgs
- Build loader based on dot icon
- Remove old unused classes
- fix mixin for visually show class
- Add no-map icon and update snapshots
- Remove close/no-map unused old classes
- Remove unused filter icon old class
- Update utilities hash
- fix(Cache.php) do not try to store strings larger than packet size
- add positioning theme override for twentytwenty
- Add mail, phone and website icons for events-pro
- Add map-pin icon for events-pro venue
- add override for svg in twentynineteen
- remove fills from icon components
- Add caret-alt icon for datepicker and other components
- doc(src/Tribe/Cache) fix typo
- Update caret buttons icon
- Add caret-alt-down icon
- add overrides for anchors in twentyseventeen
- add theme overrides for widget area in twentyseventeen
- Add messages-not-found icon
- doc(readme.txt) update changelog comment
- doc(tests/wpunit) udpate comment
- fix(Repository,Utils/Array) correctly handle query arguments merge
- doc(src) add missing doc block
- Update loader to be compatible with the accent color override
- add anchor overrides for enfold
- Update class name for specific svg shapes
- Add play icon and update snapshots
- Add video and virtual icons
- fix(Cache) do not cache and return the serialized value
- Add arrow-right icon
- Remove redundant height and let the viewBox set it
- Fix format and a couple typos.
- Add function to handle converting an array to dependency attributes - used on v2 widgets.
- Fix messages not found color
- wip - explore control customization
- CR updates
- Add specificity to caret button
- build(vendor/momentjs) update bundled momentjs to 2.19.3
- Remove caret-alt icons
- fix(Rewrite.php) correctly check for sub-directory installations
- Add missing error icon
- Move play icon fill outside of tec (will be in VE)
- Point utilities package back to main
- feat(Utils/Path) added the class and merge method
- refactor(Template) use Paths::merge to merge paths
- fix(Rewrite) use case-insensitive replacement to build the canonical URL
- Update btn icon class name
- Remove height from loader dot icon
- Remove btn icon svg caret left/right modifiers
- Use newly added opacity value
- remove unused search and filter styles
- Address code review suggestions.
- save before push...
- CR tweaks
- Add some tests
- Handle quotes better.
- doc(readme.txt) add changelog entry
- Missed a comment
- remove theme class from theme override
- remove them class from enfold theme override
- update tests
- change var name
- update utilities hash
- Update syntax `array()` to `[]`
- Update syntax `array()` to `[]`
- Update syntax `array()` to `[]`
- Clean up formatting & add ticket refs for todos.
- Update syntax `array()` to `[]`
- Remove deprecated code
- Adding a workflow to perform submodule synchronization
- :fast_forward: https://github.com/moderntribe/tribe-common/commit/1c5a00e3207bcaa78d3c7226c92043291a358710
- :fast_forward: https://github.com/moderntribe/tribe-common-styles/commit/312d08c0e5fc23f057df01611d716c9d01e3e3d2
- :fast_forward: https://github.com/moderntribe/tribe-common-styles/commit/ebf66c89d524693e8c6ecf57ee2477dee6378434
- Remove need for creating a clone directory in the GitHub workflow
- Updating hash
- Fix indentation levels
- wip - add custom Heading control
- fix(Customizer/Controls) udpate the logic and setting for the Heading control
- Add styles to h4
- Fix sintax for PHP 5.6
- Fix syntax errors of PHP version
- Fix sintax errors
- Fix submodule hash
- add full common styles to stylesheet array, add widget inline styles check
- fix incorrect class for plus icon
- update snapshots for plus icon
- add sensible defaults for skeleton icons
- code review fixes
